test/util_test.js

```diff
@@ -223,33 +257,46 @@ describe('importData()', function() {
     const path = sinon.spy();
     const imports = sinon.stub().returns();
 
-    const mockClient = sinon.stub().returns({
+    const clientMock = sinon.stub().returns({
       datasetPath: path,
       importData: imports,
     });
 
-    const autoMlMock = {v1beta1: {AutoMlClient: mockClient}};
+    const autoMlMock = {v1beta1: {AutoMlClient: clientMock}};
+
+    const settingsMock = {
+      secretToken: 'foo',
+    };
+
     const util = proxyquire('../src/util.js', {
       '@google-cloud/automl': autoMlMock,
+      '../functions/settings.json': settingsMock,
     });
 
     util.importData(projectId, computeRegion, datasetId, file);
     sinon.assert.calledOnce(path);
     assert(path.calledWith(projectId, computeRegion, datasetId));
     sinon.assert.calledOnce(imports);
   });
+
   it('should throw an error', function() {
     const path = sinon.spy();
     const imports = sinon.stub().throws();
 
-    const mockClient = sinon.stub().returns({
+    const clientMock = sinon.stub().returns({
       datasetPath: path,
       importData: imports,
     });
 
-    const autoMlMock = {v1beta1: {AutoMlClient: mockClient}};
+    const autoMlMock = {v1beta1: {AutoMlClient: clientMock}};
```

Comment:
Nit: move this logic into a function? (e.g. `getMocks()`?)

(This is the pattern we use in our other Node repos.)

Reply:
Do you have an example of this? I'm looking through repos and not finding they way that you all do this.

Reply:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/imagemagick/test/index.test.js#L37